product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -213,16 +214,54 @@ Enable **Superuser Access** to grant superuser privileges to the edb_admin role.
+Enable **Transparent Data Encryption (TDE)** to use your own encryption key. This option is available for EDB Postgres Advanced Server and EDB Postgres Extended Server for version 15 and later. Select an encryption key from your project and region to encrypt the cluster with TDE. To learn more about TDE support, see [Transparent Data Encryption](../../overview/03_security/#your-own-encryption-key---transparent-data-encryption-tde).
 
 !!!Note "Important"
-- To enable and use TDE for cluster, the encryption key must be enabled and added at the project level. For more information, see [Adding a TDE key at project level](../../administering_cluster/projects.mdx/#adding-a-tde-key).
-- If the TDE enabled cluster creation is in progress and the process is waiting providing **Encryption Key Error** next to the cluster name on the clusters page:
-  - Select the cluster name and go to the cluster's home page.
-  - See the **Action required: grant key permissions to activate the cluster** on the cluster's overview tab.
-  - Copy the **Principal** and add it to your key policy to provide **kms:Encrypt** and **kms:Decrypt** permission. 
+- To enable and use TDE for a cluster, the encryption key must be enabled and added at the project level before creating a cluster. 
+  To add a key, see [Adding a TDE key at project level](../../administering_cluster/projects.mdx/#adding-a-tde-key).
+- To enable and use TDE for a cluster, you must complete the configuration on the platform of your key management provider after creating a cluster. See [Completing the TDE configuration](#completing-the-TDE-configuration) for more information.
 !!!
 
+#### Completing the TDE configuration
+
+After you create the cluster in the BigAnimal console, the UI will display the **Waiting for access to encryption key** state. To complete the configuration and enable the key sync between BigAnimal and the key management platform you must grant encrypt and decrypt permissions to your key:
+
+1.  In BigAnimal, select the cluster name and access the cluster's page. See the **Action required: grant key permissions to activate the cluster**.
+
+1.  Copy the **Principal** identifier (AWS), **service account** (GCP) or **MSI Workload Identity** (Azure) to your clipboard. 
+
+1.  Follow the on-screen guide to grant encrypt and decrypt permissions to your key. Here is additional information in case you require further guidance:
+
+    <details><summary>AWS</summary>
+
+    1. Copy the **Principal** identifier to your clipboard. 
+    1. Go to the AWS console, and navigate to the **Key Management Service**
+    1. Select **Customer-managed keys**, and **Edit policy** for your key.
+    1. Paste the **Principal** identifier you copied to your clipboard into the `Principal.AWS` field. 
```

Comment:
We should mention that a new statement needs to be added. The Policy should change from something like this:
```
{
    "Version": "2012-10-17",
    "Id": "key-consolepolicy-3",
    "Statement": [
        {
            "Sid": "Enable IAM User Permissions",
            "Effect": "Allow",
            "Principal": {
                "AWS": "arn:aws:iam::142348057180:root"
            },
            "Action": "kms:*",
            "Resource": "*"
        }
    ]
}
```

to something like this:
```
{
    "Version": "2012-10-17",
    "Id": "key-consolepolicy-3",
    "Statement": [
        {
            "Sid": "Enable IAM User Permissions",
            "Effect": "Allow",
            "Principal": {
                "AWS": "arn:aws:iam::142348057180:root"
            },
            "Action": "kms:*",
            "Resource": "*"
        },
        {
            "Sid": "Enable TDE on cluster p-k2nxxtndkg",
            "Effect": "Allow",
            "Principal": {
                "AWS": "arn:aws:iam::142348057180:role/ba-vcvolsnap0ZUPsgZ-us-east-1-p-k2nxxtndkg"
            },
            "Action": [
                "kms:Encrypt",
                "kms:Decrypt"
            ],
            "Resource": "*"
        }
    ]
}
```